### PR TITLE
 the default option of a select

### LINKED ISSUE #36573: What is the default option of a select element when none is selected?
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select

### What specific section or headline is this issue about?

Try It

### What information was incorrect, unhelpful, or incomplete?

> You can include a [selected](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option#selected) attribute on an <option> element to make it selected by default when the page first loads.

The above section is incomplete.
What is the default selected element when no selected default is supplied? 

### What did you expect to see?

The first element will be selected as the default if none is specified.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/select`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/select/index.md
* Last commit: https://github.com/mdn/content/commit/816cc4d4a5a318a23222946b6981bb92b499aebb
* Document last modified: 2024-10-11T12:51:34.000Z

</details>

### PR DESCRIPTION
**Pull request by jolka-ef**
Update information about the selected option.

Fixes #36573

### CODE REVIEW

File: files/en-us/web/html/element/select/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The **`<select>`** [HTML](/en-US/docs/Web/HTML) element represents a control tha
...
 
-Each `<option>` element should have a [`value`](/en-US/docs/Web/HTML/Element/option#value) attribute containing the data value to submit to the server when that option is selected. If no `value` attribute is included, the value defaults to the text contained inside the element. You can include a [`selected`](/en-US/docs/Web/HTML/Element/option#selected) attribute on an `<option>` element to make it selected by default when the page first loads.
+Each `<option>` element should have a [`value`](/en-US/docs/Web/HTML/Element/option#value) attribute containing the data value to submit to the server when that option is selected. If no `value` attribute is included, the value defaults to the text contained inside the element. You can include a [`selected`](/en-US/docs/Web/HTML/Element/option#selected) attribute on an `<option>` element to make it selected by default when the page first loads. The first element will be selected as the default if none is specified.
```
Comment (chrisdavidmills) [inline files/en-us/web/html/element/select/index.md:16]:
```suggestion
Each `<option>` element should have a [`value`](/en-US/docs/Web/HTML/Element/option#value) attribute containing the data value to submit to the server when that option is selected. If no `value` attribute is included, the value defaults to the text contained inside the element. You can include a [`selected`](/en-US/docs/Web/HTML/Element/option#selected) attribute on an `<option>` element to make it selected by default when the page first loads. If no `selected` attribute is specified, the first `<option>` element will be selected by default. If multiple `<option>` elements have `selected` attributes specified, the last one in the source order will be selected by default.
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Hi there @jolka-ef , and thank you for your contribution to MDN! What you added was fine, but I've offered a suggestion that I feel makes it a little clearer, plus I've also suggested an extra sentence that provides another bit of information that was missing. Let me know what you think!

**Comment by jolka-ef** [comment]
I think now it's much better and clearer than what I added. Thanks a lot!

**Comment by jolka-ef** [comment]
@chrisdavidmills  Before I begin my actual comment, I would like to apologize in advance for my inadequate level of English proficiency. 
I am not a native speaker :)
I think the text is now much better and clearer, although I have some doubts about this part

_**If multiple `<option>` elements have `selected` attributes specified, the last one in the source order will be selected by default.**_

because if I understand the [specification](https://html.spec.whatwg.org/multipage/form-elements.html#the-select-element) correctly, that's not usage they encourage.
Specs states 

**_A select element whose multiple attribute is not specified must not have more than one descendant option element with its selected attribute set._**

**Comment by jolka-ef** [comment]
Hi @chrisdavidmills !
I add only part of changes. Is it okay with you ?

**Comment by chrisdavidmills** [comment]
> Hi @chrisdavidmills ! I add only part of changes. Is it okay with you ?

Hi @jolka-ef! Sorry I missed this earlier. Yes, I think this is fine—the change you've made fixes the actual issue you were tackling. Thanks so much!

**Review by chrisdavidmills — APPROVED** [review]
LGTM!

**Pull request merged**